src/commands/deleteResourceGroup/deleteResourceGroup.ts

```diff
@@ -45,9 +44,6 @@ export async function deleteResourceGroup(context: IActionContext, primaryNode?:
         }
 
         void node.deleteTreeItem(context);
-        const message: string = localize('startedDelete', 'Started delete of resource group "{0}".', node.name);
-        void window.showInformationMessage(message);
-        ext.outputChannel.appendLog(message);
```

Comment:
Removed this extra notification, and then moved the appendLog call into `DeleteResourceGroupStep`